sympy/solvers/tests/test_solvers.py

```diff
@@ -1141,12 +1134,9 @@ def test__invert():
 
 def test_issue_4463():
     assert solve(-a*x + 2*x*log(x), x) == [exp(a/2)]
-    assert solve(a/x + exp(x/2), x) == [2*LambertW(-a/2)]
```

Comment:
This test seems to be removed. Please double cross check if any other tests are removed by mistake.

Reply:
How did that happened!!. Okay I will verify it.

Reply:
Done